app/controllers/v1/sessions_controller.rb

```diff
@@ -306,7 +307,8 @@ def login_stats(status, error = nil)
       when :success
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY, tags: [VERSION_TAG]) if type == 'signup'
         StatsD.increment(STATSD_LOGIN_STATUS_SUCCESS, tags:)
-        Rails.logger.info("LOGIN_STATUS_SUCCESS, tags: #{tags}")
+        log_tags = tags.dup << "icn:#{@current_user.icn}"
+        Rails.logger.info("LOGIN_STATUS_SUCCESS, tags: #{log_tags}")
         Rails.logger.info("SessionsController version:v1 login complete, user_uuid=#{@current_user&.uuid}")
```

Comment:
🤔 I wonder why the safe navigation operator is used for accessing `uuid` on `@current_user` on line 312, whereas you call `icn` on `@current_user` without it on line 310. Can `@current_user` be nil when a login has succeeded? My guess is no--in which case the safe navigation operator usage is unnecessary--but it might be worth tracking that down to verify.

Reply:
I was skeptical about that `@current_user&.uuid` as well but decided to leave it at the time; now that I'm refactoring I'll take a deeper look.

Reply:
The `@current_user` is instantiated by calling `user_session_form.persist`, which results in validations being run on the User object & the User Identity object before they're saved - I had some issues with those validations with some users and never made it to the `login_stats(:success)` block here, so I think it's okay to remove this safe navigation operator.